### PR TITLE
Use generated namespace as deployment id

### PR DESCRIPTION
**Pull request by beckmani**
If Namespace is generated us it as deployment ID as well
Otherwise, I f user set namespace in config.yaml, generate only the deployment id

### CODE REVIEW

File: tests/fixtures.py [hunk 1]
```diff
@@ -32,16 +31,21 @@ def load_config():
 
 
 @pytest.fixture(scope='session')
-def set_namespace(request, load_config):
+def session_id():
+    return random_id(length=5)
```
Comment (y0sher) [inline tests/fixtures.py:35]: wouldn't that return a different id every time called?

Comment (y0sher) [reply tests/fixtures.py:35]: if it happens only as a fixture and it's scoped to a session then only once a session right?

Comment (beckmani) [reply tests/fixtures.py:35]: Yes. But its scope is session, so it is called only once per session

File: tests/test_bs.py [hunk 2]
```diff
@@ -369,7 +374,7 @@ def test_bootstrap(set_namespace, setup_bootstrap):
 def test_client(set_namespace, setup_clients, save_log_on_exit):
```
Comment (y0sher) [inline tests/test_bs.py:374]: I don't get why set_namespace is called here and not in the same way session_id is set. (like in setup_bootstrap)

**Review by y0sher — APPROVED** [review]

**Pull request merged**